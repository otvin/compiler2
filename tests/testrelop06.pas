program testrelop06(output);
{relational operators with character types}

var c,d:char;

begin
	c := 'A';
	d := 'C';

	if c < d then
		writeln('A is less than C');

	if c = d then
		writeln('oops')
	else
		writeln('A is not equal to C');

	d := pred(pred(pred(d)));

	writeln ('d now equals ',d);

	if d < c then 
		writeln(d, ' is now less than ', c);

end.
