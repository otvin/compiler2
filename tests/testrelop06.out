A is less than C
A is not equal to C
d now equals @
@ is now less than A
